rmgpy/cantherm/pdep.py

```diff
@@ -629,13 +629,15 @@ def saveInputFile(self, path):
                 f.write('    products = [{0}],\n'.format(', '.join([repr(str(spec)) for spec in rxn.products])))
                 f.write('    transitionState = {0!r},\n'.format(rxn.transitionState.label))
                 if rxn.kinetics is not None:
+                    if isinstance(rxn, LibraryReaction):
+                        if 'Reaction library:' not in rxn.kinetics.comment:
+                            rxn.kinetics.comment += 'Reaction library: {0!r}'.format(rxn.library)
+                    else:
+                        if 'Reaction family:' not in rxn.kinetics.comment:
```

Comment:
So with proper seed mechanisms I actually added a kinetics comment that notes the family the kinetics object came from.  So I think it's redundant to add the reaction family.  (I didn't however add the library so the library comment still needs added)

Reply:
Will it also output the family if seed generation is turned off?
If we both use the same syntax, we can keep the code here since it first checks whether the comment exists or not

Reply:
I am unsure the status of the relationship between proper seed mechanisms and all our other outputs. Will this break the seed mechanisms? 

Is this redundant in chemkin output or other output formats or just the seed mechanisms?

Reply:
It's added to the kinetics comment when it is made so it is entirely independent of seed mechanism generation (I just needed something to parse the family out of).  This means it will be redundant in all cases.  

Since this is appended to the kinetics comment when the input network file is saved the comment would only ever be parsed if the network file was loaded and the comment parsed.  I'm not aware of anywhere that happens.  If I am correct and that doesn't happen anywhere I wouldn't expect this to break anything.

Reply:
There's also no point in tagging it the same way we do for seed mechanisms because the network file format is different. So this should be fine.

Reply:
The comment I added is just 'family:', and it is always present if it was made from a family so your catch both won't work and is unnecessary.